docs/markdown/renderer.js

```diff
@@ -97,7 +97,7 @@ const Heading = props => {
   }
   const css = {
     marginTop: 0,
-    '&:not(:first-child)': { marginTop: 30 },
+    '&:not(:first-of-type)': { marginTop: 30 },
```

Comment:
Is there an open issue we could link for this fix?

Reply:
I guess no. I found it in the console panel